open-ce/docker_build.py

```diff
@@ -21,6 +24,15 @@
 
 DOCKER_TOOL = "docker"
 
+def make_parser():
+    ''' Parser for input arguments '''
+    arguments = [utils.Argument.DOCKER_BUILD]
+    parser = utils.make_parser(arguments,
+                               description='Run Open-CE tools within a container',
+                               formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
`make_parser` defaults to the formatter from utils. Then you wouldn't have to import argparse.

```suggestion
    parser = utils.make_parser(arguments, description='Run Open-CE tools within a container')
```